src/test/mockResultTableStore.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Log } from 'sarif';
+
+export const log = {
```

Comment:
Reason for having the whole log? Only the `results` seem to be used.

Reply:
pushing a fix in this PR: https://github.com/microsoft/sarif-vscode-extension/pull/304 👍